x-pack/plugins/security_solution/server/endpoint/routes/resolver/queries/multi_searcher.ts

```diff
@@ -34,6 +34,10 @@ export interface QueryInfo {
    * one or many unique identifiers to be searched for in this query
    */
   ids: string | string[];
+  /**
+   * a function to handle the response
+   */
+  handler: (response: SearchResponse<ResolverEvent>) => void;
```

Comment:
This calls into the class (e.g. RelatedEventsQueryHandler etc) for it to handle the response from ES.